src/plugins/custom/repository.ts

```diff
@@ -48,10 +68,11 @@ function createRepository (fastify: FastifyInstance) {
     },
 
     findMany: async <T>(table: string, opts: QueryOptions = {}): Promise<T[]> => {
-      const { select = '*', where = { 1: 1 } } = opts
+      const { select = '*', where = { 1: 1 }, join } = opts
       const [clause, values] = processAssignmentRecord(where, 'AND')
+      const joinClause = buildJoinClause(join)
 
-      const query = `SELECT ${select} FROM ${table} WHERE ${clause}`
+      const query = `SELECT ${select} FROM ${table} ${joinClause} WHERE ${clause}`
```

Comment:
General question:

Are these sanitized btw? To prevent SQL injections?

Why not use [knex](https://knexjs.org/) for this whole part?

Reply:
> Why not use [knex](https://knexjs.org/) for this whole part?

Ok